teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization)
+    super(name, age, parent_permission)
```

Comment:
- [ ] Ensure you add the other arguments which are name, age and parent_permission.